main/webapp/modules/core/scripts/facets/text-search-facet.js

```diff
@@ -141,9 +141,9 @@ TextSearchFacet.prototype._initializeUI = function() {
     this._elmts.input[0].value = this._query;
   }
   
-  this._elmts.input.bind("keyup change input",function(evt) {
-    // Ignore non-character keyup changes
-    if(evt.type === "keyup" && (this.value === self._query || this.value === '' && !self._query)) {
+  elmts.input.bind("keyup change input",function(evt) {
```

Comment:
This line throws an error:

<img width="960" alt="2020-07-01" src="https://user-images.githubusercontent.com/52909743/86229511-72eb9700-bbad-11ea-94be-c76f01f17e1e.png">

If I switch it back to `this._elmts`, it works just fine and the `next` and `last` buttons work as well.

Reply:
Fixed.